ceph/cephfs/README.md

```diff
@@ -48,6 +48,17 @@ kubectl create -f claim.yaml
 kubectl create -f test-pod.yaml
 ```
 
+* Provision for claims in specific namespaces
+
+If only provisioning for specific namespaces, start the provisioner with `-namespace=target_namespace`, e.g
```

Comment:
can we accept more than one here ? if not 's/specific namespaces/specific namespace'